db/migrate/20230822184809_create_posts.rb

```diff
@@ -0,0 +1,13 @@
+class CreatePosts < ActiveRecord::Migration[7.0]
+  def change
+    create_table :posts do |t|
+      t.text :Text
+      t.datetime :CreatedAt
+      t.datetime :UpdatedAt
+      t.integer :CommentsCounter, default: 0
+      t.integer :LikesCounter, default: 0
+
+      t.timestamps
```

Comment:
- [ ] [Optional] [Highly recommended] Please remove columns `:CreatedAt` and `:UpdatedAt` because `t.timestamps` at the end will create it for you automatically.

- [ ] Same here, You should use snake_case for naming database columns according to the [Rails naming conventions](https://gist.github.com/iangreenleaf/b206d09c587e8fc6399e#database). Please name these columns `:comments_counter` and `:likes_counter`, instead of `:CommentsCounter` and `:LikesCounter`.